coordinator/src/position/models.rs

```diff
@@ -87,6 +88,14 @@ pub struct Position {
 
     pub position_state: PositionState,
 
+    /// Accumulated order matching fees for the lifetime of the position.
+    ///
+    /// We use an [`Option`] to maintain backwards-compatibility with older positions. This is
+    /// important because we cannot safely resize old positions, since we do not know the accrued
+    /// order matching fees. Without this information the coordinator could end up losing these
+    /// fees.
+    pub order_matching_fees: Option<Amount>,
```

Comment:
I don't think we need to use an option here if we write a simple migration script. The position contains all data necessary to calculate the `order_matching_fees`.